src/operators/index.ts

```diff
@@ -1,4 +1,3 @@
 export * from './derive.operator';
 export * from './omit.operator';
-export * from './null-to-undefined.operator';
```

Comment:
This was only used by a `@Transform` call that is now in `class-decorators`. The other operators are all used for shared types magic to pick/omit decorated fields.